packages/cli/src/commands/generate/deploy/providers/aws_serverless.js

```diff
@@ -0,0 +1,98 @@
+import path from 'path'
+import fs from 'fs'
+
+import { getPaths } from 'src/lib'
+
+export const apiProxyPath = '/api'
+
+const SERVERLESS_YML = `# See the full yml reference at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: app
+
+# Uncomment org and app if you want to integrate your deployment with the serverless dashboard. See https://www.serverless.com/framework/docs/dashboard/ for more details.
+# org: your-org
+# app: your-app
+
+plugins:
+  - serverless-dotenv-plugin
+
+custom:
+  dotenv:
+    include:
+      - # List the environment variables you want to include from your .env file here.
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-2 # This is the AWS region where the service will be deployed.
+  httpApi: # HTTP API is used by default. To learn about the available options in API Gateway, see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html
+    cors: true
+    payload: '1.0'
+  stackTags: # Add CloudFormation stack tags here
+    source: serverless
+    name: Redwood Lambda API with HTTP API Gateway
+  tags: # Add service wide tags here
+    name: Redwood Lambda API with HTTP API Gateway
+
+package:
+  individually: true
+
+functions:
+  ${fs
+    .readdirSync(path.resolve(getPaths().api.functions))
+    .map((file) => {
+      const basename = path.basename(file, '.js')
+      return `${basename}:
+    description: ${basename} function deployed on AWS Lambda
+    package:
+      artifact: dist/zipball/${basename}.zip # This is the default location of the zip file generated during the deploy command.
```

Comment:
```suggestion
      artifact: api/dist/zipball/${basename}.zip # This is the default location of the zip file generated during the deploy command.
```

If we place the config in the root directory, then I guess we'll have to modify this as well.